ProcessSlave.php

```diff
@@ -389,9 +389,9 @@ protected function prepareEnvironment(\React\Http\Request $request)
             $_SERVER['HTTP_' . strtoupper(str_replace('-', '_', $name))] = $value;
         }
 
-        //HTTP_X_REAL_IP is always set, either through ProcessMaster or by a proxy
-        //in front of ProcessMaster.
-        $_SERVER['REMOTE_ADDR'] = $_SERVER['HTTP_X_REAL_IP'];
+        //Using ProcessManager in front of the slaves masks the real/proxy ip
+        $_SERVER['REMOTE_ADDR'] = $_SERVER['HTTP_X_REMOTE_IP'];
```

Comment:
Since injectHeader does not overwrite incoming header, this could lead to a sec vulvn. I'm currently working on it to overwrite that header always by `incoming->getRemoteAddress`, so a client can not set it.